dev/ory/kratos.yml

```diff
@@ -77,11 +77,23 @@ selfservice:
       after:
         password:
           hooks:
+            - hook: web_hook
+              config:
+                url: http://e2e-tests:4012/kratos/registration
```

Comment:
this needs to point out to the api server instead of `e2e-tests`

Reply:
Does this need to be the public endpoint or the k8s internal service endpoint of the api server?

Reply:
should be the internal endpoint only. 

this endpoint should not be publicly accessible

Reply:
although probably a big deal if this is (for now) because it's protected by a key

Reply:
Will the pod that connects to this endpoint be in the same namespace as the api server? We can hardcode it as `api:4012/kratos/registration` if so, no templating needed. Otherwise will have to include the fqdn - `api.namespace.svc.cluster.local:4012/kratos/registration` and that will require templating.

Reply:
kratos will connect to it. afaik it's in the same namespace?